### PR TITLE
feat: In-context translating on production TG-459

### PR DESCRIPTION
**Pull request by stepan662**

### CODE REVIEW

File: docs/web/in_context.mdx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+![In-context translating](/img/in-context.gif)
+
+## In-context on production
+
+To enable in-context translating on production you can pass `apiKey` and `apiUrl` through [Tolgee Tools](https://chrome.google.com/webstore/detail/tolgee-tools/hacnbapajkkfohnonhbmegojnddagfnj) plugin, that way Tolgee will switch to development mode (plugin will also supply `@tolgee/ui` package automatically).
+
+:::warning
+Never include `apiKey` in Tolgee configuration on production.
+This property is intended only for local development, as it's not ment to be publicly visible.
```
Comment (marketachalupnikova) [inline docs/web/in_context.mdx:24]: meant

File: docs/web/in_context.mdx [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+ 5. You can edit translations (changes will get stored to Tolgee platform)
+
+![In-context translating](/img/in-context.gif)
+
+## In-context on production
+
+To enable in-context translating on production you can pass `apiKey` and `apiUrl` through [Tolgee Tools](https://chrome.google.com/webstore/detail/tolgee-tools/hacnbapajkkfohnonhbmegojnddagfnj) plugin, that way Tolgee will switch to development mode (plugin will also supply `@tolgee/ui` package automatically).
+
+:::warning
+Never include `apiKey` in Tolgee configuration on production.
```
Comment (JanCizmar) [inline docs/web/in_context.mdx:23]: I would be more explanatory here. As a newbie, I don't know what you mean by  "Tolgee configuration on production" and I would probably think that I shouldn't do exactly what you described in the paragraph before.

Comment (stepan662) [reply docs/web/in_context.mdx:23]: I think it's understandable in context of previous part which explains in-context in development mode

Comment (stepan662) [reply docs/web/in_context.mdx:23]: For local development, supply `apiUrl`,`apiKey` and `ui` to Tolgee configuration (see [Integration](/docs/platform/integration)). If you set everything properly, you can use in-context translating:

**Pull request merged**